<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiMainWindow</class>
 <widget class="QMainWindow" name="GuiMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">GuiMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBoxFileName">
      <property name="title">
       <string>File name</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButtonRecentFiles">
         <property name="text">
          <string notr="true">&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditFileName"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOpenFile">
         <property name="text">
          <string notr="true">...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="FormatsWidget" name="widgetFormats" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBoxAdvanced">
          <property name="text">
           <string>Advanced</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxExtra">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonDemangle">
             <property name="text">
              <string>Demangle</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonShortcuts">
             <property name="text">
              <string>Shortcuts</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonOptions">
             <property name="text">
              <string>Options</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAbout">
             <property name="text">
              <string>About</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonExit">
             <property name="text">
              <string>Exit</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormatsWidget</class>
   <extends>QWidget</extends>
   <header>formatswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
